pytket/docs/changelog.rst

```diff
@@ -7,6 +7,8 @@ Unreleased
 Features:
 
 * Enabling pickling of `Architecture` objects.
+* Added optional inclusive filters to the :py:meth:`DecomposeBoxes` pass to 
```

Comment:
Minor thing but I think if you do :py:class`~pytket.passes.DecomposeBoxes` the crosslink to the docs should be generated

Reply:
Thanks for the hint, I have done it with :py:meth: instead to get the method formatting. Let me know if you think it's worth setting up a chore task at some point to go through the docs and make the crosslinks for any class or method reference currently in the docs